subprojects/docs/src/docs/userguide/tutorial/part4_gradle_plugins.adoc

```diff
@@ -0,0 +1,144 @@
+- Apply specific configuration (e.g. add organisational repositories or enforce standards).
+- Add new properties and methods to existing type via extensions.
+
+Our build file currently contains the `application` as the plugin:
+[source]
+----
+plugins {
+    // Apply the application plugin to add support for building a CLI application in Java.
+    application
+}
+----
+
+There is a plugin that is maintained and distributed by Gradle called the <<distribution_plugin#distribution_plugin,Distribution Plugin>>.
+The Distribution Plugin facilitates building archives that serve as distributions of the project.
+
+Apply the plugin by adding `distribution` to the plugins block in `build.gradle.kts`:
+[source]
+----
+plugins {
+    // Apply the application plugin to add support for building a CLI application in Java.
+    application
+    distribution
+}
+----
+
+Don't forget to sync Gradle if you are using IntelliJ IDEA.
+
+Let's make sure the plugin has been applied by looking at the new tasks it has created.
+
+Run `./gradlew :app:tasks` in your command line:
+[source]
+----
+./gradlew :app:tasks
+
+> Task :app:tasks
+
+------------------------------------------------------------
+Tasks runnable from project ':app'
+------------------------------------------------------------
+
+...
+
+Distribution tasks
+------------------
+assembleDist - Assembles the main distributions
+distTar - Bundles the project as a distribution.
+distZip - Bundles the project as a distribution.
+installDist - Installs the project as a distribution as-is.
+----
+
+A new set of distribution tasks are now available called `assembleDist`, `distTar`, `distZip` and `installDist`.
+
+Similarly, the new tasks from the Distribution plugin are now available in IntelliJ in the Gradle right-hand pane.
+
+image::tutorial/intellij-idea-plugin.png[]
+
+== Step 2. Use the Plugin
+To use the plugin, simple run the `assembleDist` task by running `./gradlew :app:assembleDist`.
+[source]
+----
+./gradlew :app:assembleDist
+
+BUILD SUCCESSFUL in 577ms
+----
+
+The `assembleDist` task builds the project and creates a `app.tar` and `app.zip` distribution in the `build/distributions` directory.
+
+image::tutorial/intellij-idea-dist.png[]
+
+== Step 3. Distribute the App
+A TAR or ZIP file can be used internally to distribute the app amount peers.
+
+A distribution can also be published to the public using the <<publishing_ivy.adoc#publishing_ivy, Ivy Publish Plugin>> or the <<publishing_maven.adoc#publishing_maven, Maven Publish Plugin>>.
+
+For example, if you wanted to publish your new app to an Ivy repository with the <<publishing_ivy.adoc#publishing_ivy, Ivy Publish Plugin>>, you would modify your build script as follows:
+[source]
+----
+plugins {
+    `ivy-publish`
+}
+
+publishing {
+    publications {
+        create<IvyPublication>("myDistribution") {
+            artifact(tasks.distZip.get())
+            artifact(tasks["customDistTar"])
+        }
+    }
+}
```

Comment:
There is so much more to publishing, I think you should probably leave it out of this intro set of tutorials completely.